fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -154,7 +173,8 @@ async def recv(self):
     async def close(self, code: int = 1000):
         if self._websocket is not None:
             # Case opened, we have something to close.
-            self._websocket.close(code)
+            await self._websocket.close(code)
```

Comment:
can't websocket be None ?

Reply:
There's `if self._websocket is not None:" just above...